client/hostinfo_unix.cpp

```diff
@@ -39,6 +39,8 @@
 #include <vector>
 #include <string>
 #include <cstring>
+#include <tuple>
```

Comment:
I'm not sure this is available in VS2010

Reply:
Thanks, didn't know. That's annoying. So the client needs to be buildable with VS2010 on Windows? What's the motivation for supporting such an old version?

Reply:
Nevermind, I've got the VS2010 story explained to me at the workshop :) For now I'll get a workaround.

Reply:
`std::tuple` is also a C++11 extension and thus not available for the Linux part too.